### ISSUE TITLE
cli: renku status fails when no renku run was executed

### ISSUE THREAD

**Issue by dorinathanou** [SwissDataScienceCenter/renku-python#212]
Missing directory `.renku/workflows`

**Comment by dorinathanou** [comment]
```
Traceback (most recent call last):
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/bin/renku", line 11, in <module>
    sys.exit(cli())
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "//anaconda/lib/python3.5/contextlib.py", line 30, in inner
    return func(*args, **kwds)
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/renku/cli/status.py", line 57, in status
    graph = Graph(client)
  File "<attrs generated init 66c1f03b88181fd436c1aad62f20c0197dcde8f7>", line 7, in __init__
  File "/Users/thanou/.local/share/virtualenvs/myproject-2x5oeRCb/lib/python3.5/site-packages/renku/cli/_graph.py", line 55, in __attrs_post_init__
    self.client.workflow_path.resolve().relative_to(self.client.path)
  File "//anaconda/lib/python3.5/pathlib.py", line 1109, in resolve
    s = self._flavour.resolve(self)
  File "//anaconda/lib/python3.5/pathlib.py", line 330, in resolve
    return _resolve(base, str(path)) or sep
  File "//anaconda/lib/python3.5/pathlib.py", line 315, in _resolve
    target = accessor.readlink(newpath)
  File "//anaconda/lib/python3.5/pathlib.py", line 422, in readlink
    return os.readlink(path)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/thanou/Documents/SDSC/renku_tutorial/sigh/.renku/workflow'
```

**Comment by jirikuncar** [comment]
closed via f0127aa3
